src/modules/rain.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```

Comment:
- [ ] Linting is excellent for tracking errors and correcting them,  so it is better not to disable them